tests/e2e/docker/hermes.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM informalsystems/hermes:0.13.0 AS hermes-builder
+FROM osmolabs/hermes:0.13.0 AS hermes-builder
```

Comment:
Why would we host our own `hermes` image?

I think we should only store `hermes-e2e` image because this is something with regards to the e2e tests that are "under our control". We also have Dockerfile for building the `hermes-e2e` image in this repository.

On the other hand, we are not developing or building hermes itself. So I think we should just reuse their image. Then, when there is a new version, we can just bump it up here. If we store our own, we have to upload a new one to our docker hub every time, and I don't really see a reason for doing that.

Let me know what you think

Reply:
Since we don't need to build the `hermes-e2e` image anymore we can use the official `hermes` image.

If for whatever reason in the future we need to deploy a specific `hermes-e2e` image for our e2e tests we can undo this change but I would try to stay as close to upstream as possible.

Regarding the other point. `osmolabs/hermes` it's just a mirror of the official `hermes` page. 
We could use the upstream one ofc but it's nice to have control of the artifacts that we use (e.g. they make the repo private / it is inaccessible / etc...). The burden of mantaining the mirror is minimal I think